Formula/toml-mode.rb

```diff
@@ -0,0 +1,28 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+SOURCE = "toml-mode-20150818.120.el".freeze
+
+class TomlMode < EmacsFormula
+  desc "Emacs Mojor mode for editing TOML files"
+  homepage "https://github.com/dryman/toml-mode.el"
+  url "https://melpa.org/packages/#{SOURCE}"
```

Comment:
We shouldn't be downloading directly from MELPA; not only do they not archive old versions but it may mess with their stats.  Could you ask the developer to tag a release on GitHub?